packages/expo-router/src/ui/common.tsx

```diff
@@ -16,10 +16,6 @@ export const ViewSlot = Slot as React.ForwardRefExoticComponent<
   ViewProps & React.RefAttributes<View>
```

Comment:
didn't see you filed one too 😅 I'm actually not even sure why we pass through a raw `View` as a ref rather than just `Component<ViewProps>` and mark it as generic.. but I'm also not sure if this is even exposed to users
https://github.com/expo/expo/pull/38899